libsignal-service/src/account_manager.rs

```diff
@@ -127,13 +145,36 @@ impl<Service: PushService> AccountManager<Service> {
             protocol_store
                 .save_pre_key(pre_key_id, &pre_key_record, None)
                 .await?;
+            // TODO: Shouldn't this also remove the previous pre-keys from storage?
```

Comment:
this is actually a very good point, I'm actually not sure what happens then.

Reply:
Will be the same revamp-pre-key issue.